config/main.yml

```diff
@@ -105,3 +105,9 @@ core:
     includeSelf: true
 misc:
   ignore-ca: false
+
+commands:
+  lookup:
+    # override set of regexes used by each command
```

Comment:
you can only override the lookup command, right? The comment suggests this works for other commands as well ...

Reply:
will update the comment.